### ISSUE TITLE
Different behavior for different end of line characters

### ISSUE THREAD

**Issue by vasilii-kovalev** [ai/check-dts#19]
The library throws "Wrong error" error for CRLF end of line (EOL) characters, but works fine for LF ones in the same test cases.

Steps to reproduce:
1. Create `test/index.errors.ts` and write cases, that should throw an error. For example:
    ```typescript
    // THROWS Type 'string' is not assignable to type 'number'.
    const variable: number = "variable";
    ```
2. Make sure that file has CRLF EOL.
3. Run `check-dts`.
    Actual behavior:
    ```
    ✖ src\tests\index.errors.ts:2:7: Wrong error
      Expected: Type 'string' is not assignable to type 'number'.
      Got: Type 'string' is not assignable to type 'number'.
    ```

    Expected behavior:
    ```
    ✔ src\tests\index.errors.ts
    ```
4. Change CRLF EOL to LF one.
5. Run `check-dts`.
    Actual behavior:
    ```
    ✔ src\tests\index.errors.ts
    ```

**Comment by ai** [comment]
Could be a bug in message formatter.

Can you send PR?

**Comment by vasilii-kovalev** [comment]
Yes, I'll try to fix it today or tomorrow.

**Comment by vasilii-kovalev** [comment]
@ai created a PR to fix the issue.

**Comment by ai** [comment]
Thanks. The fix was released in 0.5.2.
